flocker/acceptance/node_scripts/enable_service.py

```diff
@@ -0,0 +1,20 @@
+"""
+Start a service and enable across reboots.
+"""
+
+import os
+import sys
+from subprocess import check_call
+
+service = sys.argv[1]
+
+if os.path.exists("/etc/redhat-release"):
+    # Redhat-based system:
+    check_call(["systemctl", "enable", service])
+    check_call(["systemctl", "start", service])
+else:
+    # Ubuntu 14.04
+    override = "/etc/init/%s.override" % (service,)
+    if os.path.exists(override):
+        os.remove(override)
+    check_call(["service", service, "start"])
```

Comment:
This looks good to me. Specifically it looks like it will fail and exit in an error condition if it is run on a system that either doesn't have `service` installed or doesn't support `systemctl` or the `service` command. Exitting on the error condition should make ssh exit in an error condition, and consequently should result in run_ssh calling its errback, and the test should error out in that case.